test/tests/pbs_smoketest.py

```diff
@@ -135,27 +138,35 @@ def test_standing_reservation_stale_nodes(self):
         else:
             self.logger.info('Missing timezone, using America/Los_Angeles')
             tzone = 'America/Los_Angeles'
-       	a = {'resources_available.ncpus': 1}
-       	self.server.create_vnodes('vn', a, num=4, mom=self.mom)
-       	a = {'Resource_List.select': '1:ncpus=2',
+            
+        a = {'resources_available.ncpus': 1}
+        time_now = int(time.time())
+        self.server.create_vnodes('vn', a, num=4, mom=self.mom)
+        a = {'Resource_List.select': '1:ncpus=2',
              ATTR_resv_rrule: 'FREQ=HOURLY;COUNT=2',
              ATTR_resv_timezone: tzone,
              ATTR_resv_standing: '1',
-             'reserve_start': time.time() + 4,
-             'reserve_end': time.time() + 9, }
-       	r = Reservation(TEST_USER, a)
+             'reserve_start': time_now + 8,
+             'reserve_end': time_now + 14, 
+             'Resource_List.place': 'free'}
+        r = Reservation(TEST_USER, a)
+        #Submitting recurring reservation to check the resv status for one of the vnode being stale
         rid = self.server.submit(r)
+        #To check the standing resv once confirmed.
         a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2")}
-        a2 = {'reserve_state': (MATCH_RE, "RESV_DEGRADED|2")}
+        #To check the standing resv for Degraded status when one of the vnode becomes down
+        a2 = {'reserve_state': (MATCH_RE, "RESV_DEGRADED|10")}
         self.server.expect(RESV, a, id=rid)
         self.server.status(RESV, 'resv_nodes', id=rid)
         resv_node = self.server.reservations[rid].get_vnodes()[0]
+        #To check the status of standing resv in next cycle and also check the down node doesn't exist in resv_nodes.
```

Comment:
rephrase it as "Check that standing reservation gets confirmed in next cycle to a different node than the one which is down"